### DISCUSSION TITLE
Version 1.10.0

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#1586] [Changelog]
## Bugfixes
- #1555 pause incorrectly pauses stack timer blocks. (prio-3)
- #1555 pause incorrectly pauses edge-activated blocks with obscured shadows. (prio-4)
- #1555 pause discards old threads when a new thread is started while in use. (prio-3)
- #1555 pause does not stop mouse and keyboard event blocks from firing. (prio-3)
- #1555 pause breaks when sa-pause block is executed when paused. (prio-3)
- #1563 editor-devtools highlighting conflicts with editor-theme3. (prio-4)
- #1570 dark-www does not make crash screen dark. (prio-4)
- #1571 mediarecorder breaks when the user exits editor view. (prio-3)
- #1583 dark-www makes studio follower counts invisible. (prio-4)
- #1585 scratch-messaging adds asterisks to links to ST members. (prio-3)
- #1602 infinite-scrolling is incompatible with scratchr2. (prio-4)
- #1606 editor-dark-mode makes monitor names invisible. (prio-4)
- #1608 image-uploader tries to run on pages without post form. (prio-4)
- #1612 dark-www does not make popups dark. (prio-3)
- #1616 Addons with permission requirements cannot be enabled for the first time. (prio-2)
- #1623 cat-blocks does not add extra space in block palette. (prio-4)

## New features
- #1502 New addon: copy-message-link (prio-3)
- #1568 New addon: load-extensions (prio-3)
- #1597 Addons can be searched using ID. (prio-4)

**Comment by cobaltt7** [comment]
> * #1563 editor-devtools highlighting conflicts with editor-devtools. (prio-4)

theme3 highlighting conflicts with editor-devtools

**Comment by nexensys** [comment]
Suggestion: Maybe an addon that will allow you to add code by typing... something like scratchblocks but able to inject the rendered blocks into the workspace. Also maybe have different languages like the normal scratchblocks script, javascript, python, etc. I would make this myself but I can't figure out how to add the blocks to the workspace.

**Reply by mxmou** [reply]
Something similar to Snap!'s keyboard editing would be great, but I think this should be a separate issue or discussion.

**Reply by nexensys** [reply]
Okay, I'll make a new one...

**Reply by nexensys** [reply]
Here we go: #1595

**Comment by ghost** [comment]
Is Scratch Addons compatible with Edge?

**Reply by mxmou** [reply]
Yes, Edge is based on Chromium so all Chrome extensions should be compatible with it.
